engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql

```diff
@@ -652,3 +652,5 @@ INSERT INTO `cloud`.`user_vm_details`(`vm_id`, `name`, `value`)
         INNER JOIN `cloud`.`vm_instance` ON vm_instance.id = user_vm_details.vm_id
         WHERE ssh_keypairs.account_id = vm_instance.account_id;
 
+-- PR #6080 Change column `value` size from 255 to 4096 characters, matching the API "updateConfiguration" "value" size
+ALTER TABLE `cloud`.`account_details` MODIFY `value` VARCHAR(4096);
```

Comment:
When calling `MODIFY` command, it removes all the column metadata and only adds what is specified in the command (in this case, only `VARCHAR(4096)`. Currently this field is `NOT NULL`, therefore it must be informed in the command:

```suggestion
ALTER TABLE `cloud`.`account_details` MODIFY `value` VARCHAR(4096) NOT NULL;
```

Reply:
Thanks @GutoVeronezi!